README.md

```diff
@@ -19,6 +19,16 @@ Please see [our website] for more in-depth information.
 
 <img width="700px" src="./demo/manage-clusters.svg">
 
+## Why KWOK?
+
+- Lightweight: You can simulate thousands of nodes on your laptop without significant consumption of CPU or memory resources.
```

Comment:
```suggestion
- Lightweight: You can [simulate thousands of nodes] on your laptop without significant consumption of CPU or memory resources.
```
[simulate thousands of nodes]: {{< relref "/docs/user/kwok-manage-nodes-and-pods" >}}

Reply:
I have no idea whether this will work, but updated as advised. And I'll take a review then.

Reply:
Actually this doesn't work. see https://deploy-preview-451--k8s-kwok.netlify.app/

Reply:
Fixed